src/components/ui/icons/LogoIcon.tsx

```diff
@@ -0,0 +1,26 @@
+    <path d="m83.212 21.313c-1.1624 0-2.0271-0.3-2.5941-0.8998-0.5529-0.5999-0.8293-1.4159-0.8293-2.4482v-4.6036h-1.4246v-2.1762h1.4246v-2.1343l2.8705-0.837v2.9713h2.5516l-0.1701 2.1762h-2.3815v4.4152c0 0.5441 0.1276 0.9207 0.3828 1.13 0.2551 0.1953 0.652 0.2929 1.1907 0.2929 0.3969 0 0.808-0.0697 1.2333-0.2092v1.946c-0.3119 0.1256-0.6521 0.2162-1.0207 0.272-0.3685 0.0698-0.7796 0.1047-1.2332 0.1047z" />
+    <path d="m90.66 21.313c-1.2049 0-2.1475-0.3279-2.828-0.9835-0.6662-0.6696-0.9993-1.6461-0.9993-2.9295v-6.2148h2.8705v6.152c0 0.6696 0.163 1.1579 0.4891 1.4648 0.326 0.2929 0.815 0.4394 1.4671 0.4394 0.4962 0 0.9498-0.0977 1.3609-0.293 0.4252-0.1953 0.7867-0.4394 1.0844-0.7323v-7.0309h2.8705v9.9185h-2.6579l-0.1063-0.9416c-0.4253 0.3348-0.9285 0.6138-1.5097 0.837-0.567 0.2092-1.2474 0.3139-2.0413 0.3139z" />
+    <path d="m100.74 9.3645v-2.2181h3.126v2.2181h-3.126zm-0.361 16.635c-0.8932 0-1.6587-0.1256-2.2965-0.3767v-1.946c0.2268 0.0698 0.4465 0.1186 0.6591 0.1465 0.2268 0.0418 0.4324 0.0628 0.6166 0.0628 0.5812 0 0.9998-0.1186 1.2548-0.3558 0.255-0.2232 0.383-0.5859 0.383-1.0881v-9.1652h-1.5102l0.2552-2.0925h4.125v11.53c0 0.9765-0.291 1.7646-0.872 2.3645-0.567 0.6138-1.439 0.9207-2.615 0.9207z" />
+    <path d="m110.57 21.313c-0.879 0-1.701-0.0628-2.467-0.1884-0.751-0.1255-1.403-0.3069-1.956-0.544v-2.4901c0.624 0.2511 1.297 0.4534 2.02 0.6068 0.723 0.1535 1.432 0.2302 2.126 0.2302 0.992 0 1.722-0.0907 2.19-0.272 0.468-0.1814 0.702-0.5929 0.702-1.2346 0-0.4045-0.099-0.7254-0.298-0.9626-0.198-0.2371-0.546-0.4464-1.042-0.6277-0.482-0.1953-1.169-0.4115-2.062-0.6487-1.432-0.3906-2.438-0.8788-3.02-1.4647-0.581-0.5859-0.871-1.395-0.871-2.4274 0-1.2415 0.46-2.1971 1.382-2.8667 0.921-0.66961 2.261-1.0044 4.019-1.0044 0.822 0 1.587 0.0558 2.296 0.1674s1.283 0.24413 1.722 0.39758v2.4901c-1.162-0.4325-2.374-0.64871-3.636-0.64871-0.893 0-1.594 0.09765-2.105 0.29291-0.51 0.1814-0.765 0.572-0.765 1.1718 0 0.3488 0.092 0.6278 0.276 0.8371 0.184 0.2092 0.496 0.3975 0.936 0.5649 0.453 0.1674 1.084 0.3558 1.892 0.565 1.12 0.293 1.985 0.6417 2.594 1.0463 0.61 0.4045 1.028 0.8649 1.255 1.381 0.241 0.5162 0.361 1.0951 0.361 1.7368 0 1.1718-0.453 2.1134-1.361 2.8249-0.907 0.7114-2.303 1.0672-4.188 1.0672z" />
+    <path d="m118.11 25.623v-14.438h2.488l0.149 0.9207c0.439-0.3627 0.921-0.6417 1.446-0.837 0.524-0.1953 1.141-0.2929 1.85-0.2929 1.474 0 2.608 0.3975 3.402 1.1927 0.808 0.7812 1.212 2.0646 1.212 3.8502s-0.433 3.1179-1.297 3.9967c-0.851 0.8649-2.013 1.2974-3.487 1.2974-1.134 0-2.098-0.2511-2.892-0.7533v5.0638h-2.871zm5.04-6.3821c0.85 0 1.495-0.2581 1.935-0.7742 0.439-0.5162 0.659-1.3253 0.659-2.4274 0-1.0602-0.199-1.8204-0.596-2.2808-0.396-0.4743-1.034-0.7114-1.913-0.7114s-1.63 0.2999-2.254 0.8997v4.5408c0.298 0.2232 0.609 0.4045 0.935 0.544 0.341 0.1395 0.752 0.2093 1.234 0.2093z" />
```

Comment:
Make sure you use OMGSVG for every - single - SVG. Please, use my config, I find it the best balance. I think you should already have it

Reply:
Hmm, not sure what you mean by your config, do I have it? 😅